### ISSUE TITLE
Explore: Aggregation operator query without metric/label name in the result should still render a table

### ISSUE THREAD

**Issue by s-h-a-d-o-w** [grafana/grafana#23555]
**What happened**:
Using a prometheus query such as `sum(:kube_pod_info_node_count:)`, the table panel says `0 series returned.`

**What you expected to happen**:
A table that e.g. looks like:
```
| Time  | Query                           | Value #A |
|-------|---------------------------------|----------|
| 11:55 | sum(:kube_pod_info_node_count:) | 123      |
```

**Anything else we need to know?**:
This happens because when a query like this is executed, there is no metric or label name in the response.
(Cosmetic side note: Related to this, it also says "Value #A" in the legend of the graph that actually refers to nothing which should probably be removed.)

**Environment**:
- Internal ops grafana (v7.0.0-ed3eb382pre (95e635f154))

**Comment by hugohaggmark** [comment]
Thanks for creating this issue @s-h-a-d-o-w , for future reference you should always try to categorize your issues with labels. I added some labels now but it would be great if you could double check if they make sense.

**Comment by s-h-a-d-o-w** [comment]
@hugohaggmark Thanks, will keep that in mind. Labels look good to me! :)

**Comment by s-h-a-d-o-w** [comment]
This was resolved through #24018.

![Screenshot from 2020-05-06 10-37-50](https://user-images.githubusercontent.com/16936908/81155975-ca8ecd00-8f85-11ea-88b1-f012bd411eb0.png)
